Sources/TABResourceLoader/Protocols/Resource types/NetworkResourceType.swift

```diff
@@ -62,9 +62,11 @@ public protocol NetworkResourceType {
   /**
    Convenience function that builds a URLRequest for this resource
 
+   - parameter additionalQueryParameters: Query parameters to be added to the URL
    - returns: A URLRequest or nil if the construction of the request failed
    */
-  func urlRequest() -> URLRequest?
+  
+  func urlRequest(with: [URLQueryItem]) -> URLRequest?
```

Comment:
can you also please include `additionalQueryParameters` in the function signature as the parameter just to go w/ the comment above.